lib/Modal/styles.scss

```diff
@@ -18,10 +18,10 @@ $modal-media-close-dimensions: 30px;
 
 .legacy-modal {
 	width: 600px;
-	margin: 30px auto;
```

Comment:
I removed this as we central align the modal anyways. This was causing some strange visual in storybook where the certain modals weren't vertically centred 🤔